src/bot.rs

```diff
@@ -303,37 +300,35 @@ impl Bot {
         let chat_type = db::get_chat_type(conn, chat_id).await?;
         let (webxdc, version) = db::get_webxdc_version(conn, msg.get_id()).await?;
 
-        if let Ok(request) =
-            serde_json::from_str::<WebxdcStatusUpdate<GeneralFrontendRequest>>(&update)
-        {
-            match request.payload {
-                GeneralFrontendRequest::UpdateWebxdc => {
-                    let msg = send_webxdc(context, &state, chat_id, webxdc, Some(store_message()))
-                        .await?;
-                    send_newest_updates(context, msg, &mut *state.db.acquire().await?, 0, vec![])
-                        .await?;
-                    send_update_payload_only(context, msg_id, GeneralFrontendResponse::UpdateSent)
-                        .await?;
-                    return Ok(());
-                }
-            }
+        let Ok(request) = serde_json::from_str::<WebxdcStatusUpdate>(&update) else {
+            info!(
+                "Ignoring WebXDC update: {}",
+                &update.get(..100.min(update.len())).unwrap_or_default()
+            );
+            return Ok(());
```

Comment:
Maybe we can even remove this now. I don't think logging this is needed anymore.